### PR TITLE
Feature/update subscribing

### LINKED ISSUE #2224: No effective using of data on client side:  organisation profile & dashboard page
In the process of working on #2056 issue, there were found no effective using of data on client side. Case was tested by the user who is anonym or don't have permission to edit API

### Organisation profile
- OrganizationLogo.files collection. All documents are available on the page.
-  ApiLogo.files collection.All documents are available on the page.
- OrganizationApis collection. Documents are available for private APIs when an organisation has ones.
- Users collection. There are available all documents with data about the organisation managers. Documents contain fields as `_id`, `email`, `username`. 

### Dashboard page
-  ApiLogo.files collection.All documents are available on the page.
- Apis collection. There are available correct documents with all fields but data are not used. Do we really need to a subscription of API on Dashboard page?

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #2224

Organization profile:
- [x] Update subscribing on logo
- [x] Update subscribing on cover photo

Dashboard page:
- [x] APIs logos was resolved in another PR

Branding data:
- [x] Subscribe to Site logo only once
- [x] Subscribe to Site cover photo only once at the page
- [x] Remove all redundant subscriptions to branding collection

Needless subscriptions:
- [x] addApi template (remove subscribing to Organization details)
- [x] apiMetadata template (remove subscribing to Organization details)

### CODE REVIEW

**Comment by marla-singer** [comment]
@brylie Could you please review it?

**Comment by brylie** [comment]
@marla-singer reviewing today.

File: organizations/logo/collection/server/publications.js [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+  // Convert to Mongo ObjectID
+  const objectId = new Mongo.Collection.ObjectID(organizationLogoId);
+
+  return OrganizationLogo.find({
+    _id: objectId,
+    'metadata._Resumable': { $exists: false },
+  });
+});
+
+Meteor.publish('OrganizationLogoByIds', (organizationLogoIds) => {
```
Comment (brylie) [inline organizations/logo/collection/server/publications.js]: Consider changing this to `organizationLogosById`:
- it is properly camelCase (starting with a lowercase letter)
- it is somewhat more grammatical ("get me organization logos by ID" as opposed to "get organization logo by IDs")

**Comment by brylie** [comment]
This looks great and works fine for me. One **small** change requested on the publication name `organizationLogoByIds` for consistency and readability.

**Comment by brylie** [comment]
CC'ing @frenchbread to review my commit and sign-off on the PR.

**Comment by frenchbread** [comment]
@brylie  Shouldn't it logically be `organizationLogosByIds` ?

**Comment by brylie** [comment]
@frenchbread done.

**Comment by frenchbread** [comment]
Looks good to me! 👍

**Pull request merged**